deploy/juicefs-s3-gateway.yaml

```diff
@@ -29,7 +29,7 @@ spec:
           command:
             - sh
             - -c
-            - juicefs gateway $(METAURL) $(NODE_IP):9000
```

Comment:
Why not use NODE_IP?

Reply:
Using pod ip, NODE_IP is env from podIP

```
          env:
            - name: NODE_IP
              valueFrom:
                fieldRef:
                  fieldPath: status.podIP
```

Reply:
I meant `--metrics=${NODE_IP}:9567`

Reply:
Both ${NODE_IP} and 0.0.0.0 is ok, because it's separate net namespace in container